jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -23,14 +23,23 @@ installation to the data stores you were using for your 2.19 instance.
 toc::[]
 
 ## Prerequisites
+
 . Your current CircleCI Server installation is 2.19.
 . You have taken a backup of the 2.19 instance.  If you are using external datastores, they will need to be backed up separately.
-. You have a running CircleCI Server 3.0 xref:server-3-install.adoc[installation].
+. You have a new CircleCI Server 3.0 xref:server-3-install.adoc[installation]
 . You have successfully run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[reality check] with contexts prior to starting.
 . The migration script must be run from a machine with
 - `kubectl` configured for the server 3.x instance
 - `ssh` access to the 2.19 services box
 
+### External Datastores Only
+. Backups have been taken of all external data stores
+. Postgres has been updated to version 12
+
+### Internal Datastore Only
+. You have taken a backup of the 2.19 instance.
+. You have successfully run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[reality check] on the new server 3.x instance with contexts prior to starting
```

Comment:
I think Contexts should be capitalized to be more clear

```suggestion
. You have successfully run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[reality check] on the new server 3.x instance with Contexts prior to starting
```

Reply:
Huh, I have no idea.  I will defer to @rosieyohannan

Reply:
While were are deferring - should we be capitalizing Server 3.x?

Reply:
marketing says no: https://circleci.slack.com/archives/CAT9Q7TGW/p1602788866148500?thread_ts=1602788489.148400&cid=CAT9Q7TGW

Reply:
No, server is lowercase, contexts too - from Marketing